js/test/root/teststrings.js

```diff
@@ -3590,6 +3590,43 @@ module.exports.teststrings = {
         test.equal(str.formatChoice(15), "Default items");
         test.done();
     },
+    testStringFormatChoiceCharClassesComplex_mt_MT: function(test) {
+        test.expect(2);
+        var str = new IString("0#There are no items.|one#There items are one|few#The items are few|many#The items are many|#Default items");
+        str.setLocale("mt-MT");
```

Comment:
According to http://web.stanford.edu/group/cslipublications/cslipublications/TLS/TLS9-2005/TLS9_David_Karine.pdf Maltese has singular (one), dual (two), and plural (other) categories.

Reply:
the locale data have generated based on https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html there' is `many` category